docs/requirements.txt

```diff
@@ -1 +1,5 @@
 rst.linker
+mock
```

Comment:
Well, sort-of. backports.unittest_mock has a wonderful magic power when running under pytest, but has to be explicitly invoked in other environments such as a docs build. But! Since we're able to build the docs under Python 3, I just disabled support for building docs under Python 2 and then relied on unittest.mock natively. Problem solved.